pkg/lib/localstore/ftsearch/ftsearchtantivy_test.go

```diff
@@ -15,58 +15,59 @@ import (
 	"github.com/anyproto/anytype-heart/core/wallet"
 )
 
-type fixture struct {
+type fixture2 struct {
```

Comment:
2-suffix have been written consciously, I want to remove bleve in the future, so simply duplicated the codebase